src/index.js

```diff
@@ -17,6 +17,28 @@ export function t(strings, ...exprs) {
     return strings;
 }
 
+const separator = /(\${\s*\d+\s*})/g;
+const slotIdRegexp = /\${\s*(\d+)\s*}/;
+
+export function jt(strings, ...exprs) {
+    if (strings && strings.reduce) {
+        const id = getMsgid(strings, exprs);
+        const transObj = findTransObj(currentLocale, id);
+        if (!transObj) return [id];
+
+        // splits string & capturing group into tokens
+        //
+        const translatedTokens = transObj.msgstr[0].split(separator);
+
+        return translatedTokens.map((token) => {
```

Comment:
Maybe it's better to make `separator` without capturing group:
```js
const separator = /\${\s*\d+\s*}/g;
```
After that we will have an array of strings, and can place expressions by indices:
```js
const translatedStrings = transObj.msgstr[0].split(separator);
const result = []
translatedStrings.forEach((str, i) => {
   result.push(str);
   if (exprs[i] !== undefined) {
      result.push(exprs[i]);
  }
});
```
Benefits:
1. Minus one operation with regex match.
2. No string to int coerce `+slotIdMatch[1]`

What do you think?